docs/hub/open_clip.md

```diff
@@ -0,0 +1,75 @@
+# Using OpenCLIP at Hugging Face
```

Comment:
I would also add to https://moon-ci-docs.huggingface.co/docs/hub/pr_1131/en/models-libraries